types/electra/blob_sidecar.yaml

```diff
@@ -0,0 +1,7 @@
+Electra:
+  BlobSidecars:
+    type: array
+    items:
+      $ref: "../deneb/blob_sidecar.yaml#/Deneb/BlobSidecar"
+    minItems: 0
+    maxItems: 9
```

Comment:
This API returns JSON, why is it relevant this maxItems value? Even for SSZ, in it's serialized form the max value is not relevant

Reply:
It returns both JSON and SSZ
https://github.com/ethereum/beacon-APIs/blob/aa1be259e2d8a64f451dd304913e0310779f6f80/apis/beacon/blob_sidecars/blob_sidecars.yaml#L50-L52

> Even for SSZ, in it's serialized form the max value is not relevant

I quickly mentioned this [in discord](https://discord.com/channels/595666850260713488/710831784991916072/1331233019519832065), we could just use `MAX_BLOB_COMMITMENTS_PER_BLOCK` which is 4096 as max items which is what we use in Lodestar for the list limit.

Reply:
It seems better to discard invalid data as quickly as possible, and anything past `MAX_BLOBS_PER_BLOCK_ELECTRA` is invalid. No need to build in capacity in the rest of the system to handle it, either on the client or server end, for theoretical possibilities which even if working in isolation aren't useful as part of a broader system.

Reply:
You are talking about a consumer of the REST API, not a p2p network with adversarial peers. I think `MAX_BLOB_COMMITMENTS_PER_BLOCK` is fine and we don't have to deal with forks on this type anymore

Reply:
Unclear that https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Beacon/getBlobSidecars will even be useful/salient in Fulu; the beacon API might end up with a new `getColumnSidecars` or similar endpoint. A similar discussion has been happening in the req/resp area, where the v3 Fulu-only getBlobsByRange was removed in lieu of keeping only the column one.

Reply:
we can always change this to `MAX_BLOB_COMMITMENTS_PER_BLOCK` in a follow-up PR, I don't feel strongly about this as both approaches work